qbatch

```diff
@@ -77,14 +91,27 @@ if __name__ == "__main__":
             "#PBS -d {workdir}",
             "#PBS {options}",
             "#PBS {array}",
+            "{dependencies}",
             "ARRAY_IND=PBS_ARRAYID"]).format(
             nodes=nodes,
             ppn=ppn,
             shebang=shebang,
             workdir=job_workdir,
             options=' '.join(qsub_options),
-            array=use_array and '-t 1-' + str(num_jobs) or '')
+            array=use_array and '-t 1-' + str(num_jobs) or '',
+            dependencies=matching_jobids and '#PBS -W depend=afterok:' + ':'.join(matching_jobids) or '')
     elif system == 'sge':
+        matching_jobids = []
```

Comment:
Man,  how did I miss that,  that's so much easier. Will fix.
On Dec 19, 2015 3:53 PM, "Jon Pipitone" <notifications@github.com> wrote:

> In qbatch <https://github.com/pipitone/qbatch/pull/2#discussion_r48096573>
> :
>
> >      elif system == 'sge':
> > +        matching_jobids = []
>
> Actually, SGE allows -hold_jid to have wildcards, so we don't need to do
> this ourselves.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pipitone/qbatch/pull/2/files#r48096573>.
>